### DISCUSSION TITLE
Import alias for ActionScript

### DISCUSSION THREAD

**Discussion by ghost** [airsdk/Adobe-Runtime-Support#1961] [Ideas]
The Apache Royale compiler supports this syntax:

```actionscript3
import FuByteArray = flash.utils.ByteArray;
```

I'm wondering if it's possible for AIR SDK to use the same compiler fork from Apache Royale (I could try mixing them in my environment, but would be better if it were integrated with AIR SDK). And ECMAScript 4 was also going to support this (can be tested via [AS3EvalDemo](https://github.com/Klaider/ECMAScript4/blob/master/as3-evaluators/AS3EvalDemo.swf)):

(NOTE: `namespace` declarations are already supported; however ASC doesn't identify packages like this.)

```actionscript3
namespace Fu = 'flash.utils';
var ba = new Fu::ByteArray;
```

In other words, aliasing a package (its `public` namespace). If it starts with a scheme (like `https:`, note the colon), then it's not a package alias. I'm not needing this feature right now, but would be nice to have.
